web/src/views/Home.vue

```diff
@@ -31,8 +31,25 @@ import {
   getPathFromSelected,
 } from '@/utils';
 
+// redirect to "Open JupyterLab"
+const JUPYTER_ROOT = process.env.JUPYTER_ROOT || 'http://hub.dandiarchive.org ';
```

Comment:
Where else in the code is the env var `JUPYTER_ROOT` being used? Or more generally why is this being done this way? If we're never actually injecting this environment variable, I'd say we should just hard code this for now, as that's what's essentially happening anyway if we're never setting the `JUPYTER_ROOT` environment variable.

```suggestion
const JUPYTER_ROOT = 'http://hub.dandiarchive.org';
```

Reply:
either way is ok with me.  Eventually may be there would be a way to configure for local deployments to have own local jupyterlab to connect to.  Leaving it available might simplify some testing later on. But we could always bring it back when need arises.

Reply:
Thank you and I will change it to `const JUPYTER_ROOT = 'http://hub.dandiarchive.org';`